app/bundles/EmailBundle/Controller/Api/EmailApiController.php

```diff
@@ -122,64 +122,65 @@ public function sendAction(Request $request, $id)
     public function sendLeadAction(Request $request, $id, $leadId)
     {
         $entity = $this->model->getEntity($id);
-        if (null !== $entity) {
-            if (!$this->checkEntityAccess($entity)) {
-                return $this->accessDenied();
-            }
 
-            /** @var Lead $lead */
-            $lead = $this->checkLeadAccess($leadId, 'edit');
-            if ($lead instanceof Response) {
-                return $lead;
-            }
+        if (!$entity) {
+            return $this->notFound();
+        }
 
-            $post       = $request->request->all();
-            $tokens     = (!empty($post['tokens'])) ? $post['tokens'] : [];
-            $assetsIds  = (!empty($post['assetAttachments'])) ? $post['assetAttachments'] : [];
-            $response   = ['success' => false];
+        if (!$this->checkEntityAccess($entity)) {
+            return $this->accessDenied();
+        }
 
-            $cleanTokens = [];
+        /** @var Lead $lead */
+        $lead = $this->checkLeadAccess($leadId, 'edit');
+        if ($lead instanceof Response) {
+            return $lead;
+        }
 
-            foreach ($tokens as $token => $value) {
-                $value = InputHelper::clean($value);
-                if (!preg_match('/^{.*?}$/', $token)) {
-                    $token = '{'.$token.'}';
-                }
+        $post       = $request->request->all();
+        $tokens     = (!empty($post['tokens'])) ? $post['tokens'] : [];
+        $assetsIds  = (!empty($post['assetAttachments'])) ? $post['assetAttachments'] : [];
+        $response   = ['success' => false];
 
-                $cleanTokens[$token] = $value;
-            }
+        $cleanTokens = [];
 
-            $leadFields = array_merge(['id' => $leadId], $lead->getProfileFields());
-            // Set owner_id to support the "Owner is mailer" feature
-            if ($lead->getOwner()) {
-                $leadFields['owner_id'] = $lead->getOwner()->getId();
+        foreach ($tokens as $token => $value) {
+            $value = InputHelper::html($value);
```

Comment:
This is the main change in this file. The rest is just change of indentation due to "return early" change.